src/core/components/libp2p.js

```diff
@@ -50,9 +51,7 @@ module.exports = function libp2p (self) {
         })
 
         self._libp2pNode.start((err) => {
-          if (err) {
-            return callback(err)
-          }
+          if (err) { return callback(err) }
```

Comment:
Nitpick: I prefer not having unrelated formatting changes together with real code changes.